libs/login/WordPressLoginFlow/build.gradle

```diff
@@ -31,17 +35,20 @@ android {
 }
 
 dependencies {
-    implementation ('org.wordpress:utils:1.20.3') {
+    implementation ('org.wordpress:utils:1.22') {
         exclude group: "com.android.volley"
     }
 
-    implementation 'androidx.appcompat:appcompat:1.0.2'
-    implementation 'androidx.vectordrawable:vectordrawable-animated:1.0.0'
-    implementation 'androidx.media:media:1.0.1'
+    implementation "androidx.appcompat:appcompat:$appCompatVersion"
+    implementation 'androidx.vectordrawable:vectordrawable-animated:1.1.0'
+    implementation 'androidx.media:media:1.1.0'
     implementation 'androidx.legacy:legacy-support-v13:1.0.0'
     implementation 'androidx.gridlayout:gridlayout:1.0.0'
     implementation 'com.google.android.material:material:1.1.0'
 
+    implementation "androidx.core:core-ktx:$kotlin_ktx_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
     api 'com.google.android.gms:play-services-auth:15.0.1'
```

Comment:
I see we're not bumping `play-services-auth` anymore. This is probably the right call for now and I think we can try bumping it again once we merge the changes that affect the Google Sign-In method, so we can properly test it. What do you think?

Reply:
I agree. I decided not to merge it after your suggestion but we can test it later.